### PR TITLE
Add REVEL scores and REVEL rank scores to VEP-annotation output

### PR DESCRIPTION
**Pull request by alkc**
This update adds REVEL scores and REVEL ranks cores to the VCF vep annotations.

The update has been tested by running an oncogen sample through the modified pipeline and verifying that the pipeline output for the sample can be loaded into scout.

Will also be tested by running a regular WGS sample through the pipeline.

This is a draft PR until all tests are concluded.

### CODE REVIEW

**Comment by alkc** [comment]
It might be related to scout. the function used in scout to parse variants extracts both the score and the rank score:

https://github.com/Clinical-Genomics/scout/blob/2373185c2c9d71cd1bbdaaba9add232d316612d1/scout/parse/variant/variant.py#L229C1-L238

However, only the rank score is loaded into the db when the parsed variant object is passed to the function that builds the database variant object:

https://github.com/Clinical-Genomics/scout/blob/2373185c2c9d71cd1bbdaaba9add232d316612d1/scout/build/variant/variant.py#L243C1-L247

**Comment by alkc** [comment]
resuming work on this, as our instance of scout has been upgraded to a version that fixes the REVEL score issue described above. 

## TODO:

- [ ] Run a test WGS sample
- [ ] Run a test onko sample

**Comment by alkc** [comment]
Confirmed that wgs and onco runs complete without errors and that the pipeline results can be loaded into scout.

Both REVEL and REVEL rank scores are displayed properly in scout for both assays.

File: main.nf [hunk 1]
```diff
@@ -1176,22 +1176,24 @@ process annotate_vep {
...
-		--plugin MaxEntScan,$params.MAXENTSCAN,SWA,NCSS \\
 		--fasta $params.VEP_FASTA \\
 		--dir_cache $params.VEP_CACHE \\
 		--dir_plugins $params.VEP_CACHE/Plugins \\
 		--distance 200 \\
 		-cache \\
+		--plugin CADD,$params.CADD \\
+		--plugin LoFtool \\
+		--plugin MaxEntScan,$params.MAXENTSCAN,SWA,NCSS \\
+		--plugin dbNSFP,/fs1/resources/ref/hg38/annotation_dbs/dbnsfp/dbNSFP4.3a_grch38.gz,transcript_match=1,REVEL_score,REVEL_rankscore \\
```
Comment (ViktorHy) [inline main.nf:1187]: nice, better organization

**Review by ViktorHy — APPROVED** [review]

**Comment by ViktorHy** [comment]
does scout support this now?

**Comment by alkc** [comment]
> does scout support this now?

Yes!

**Pull request merged**